examples/runners/wgpu/src/graphics.rs

```diff
@@ -104,7 +104,8 @@ async fn run(
         format: preferred_format,
         width: size.width,
         height: size.height,
-        present_mode: wgpu::PresentMode::Mailbox,
+        present_mode: wgpu::PresentMode::AutoNoVsync,
```

Comment:
This change was found by @fu5ha because macOS apparently doesn't support `Mailbox` (did this *ever* work? I think `main` might be broken too).

Reply:
Note that in the latest version of this PR, I've changed it to `AutoVsync` because `AutoNoVsync` was doing excessive redraws (thousands of frames per second).